src/runner/main.cpp

```diff
@@ -152,6 +152,7 @@ int runner(bool isProcessElevated, bool openSettings, std::string settingsWindow
             L"modules/MouseUtils/PowerToys.MouseHighlighter.dll",
             L"modules/AlwaysOnTop/PowerToys.AlwaysOnTopModuleInterface.dll",
             L"modules/MouseUtils/PowerToys.MousePointerCrosshairs.dll",
+            L"modules/PowerOCR/PowerToys.PowerOCR.dll",

```

Comment:
```suggestion
            L"modules/PowerOCR/PowerToys.PowerOCRModuleInterface.dll",

```
here's your issue with loading dll :) you're trying to load wrong one

Reply:
with this fix, OCR is ran by default as soon as it's loaded, let add some hotkey to trigger it